typescript/src/schema/proto.ts

```diff
@@ -150,7 +150,25 @@ function streaming(method: MethodDescriptorProto) {
   return undefined;
 }
 
-function pagingField(messages: MessagesMap, method: MethodDescriptorProto) {
+function pagingField(
+  messages: MessagesMap,
+  method: MethodDescriptorProto,
+  service?: ServiceDescriptorProto
+) {
+  // TODO: remove this once the next version of the Talent API is published.
+  //
+  // This is a workaround to disable auto-pagination for specifc RPCs in
+  // Talent v4beta1. The API team will make their API non-conforming in the
+  // next version.
+  //
+  // This should not be done for any other API.
+  const serviceName =
+    service && service.packageName === 'google.cloud.talent.v4beta1';
+  const methodName = method.name === 'SearchProfiles' || 'SearchJobs';
```

Comment:
This won't work! It will be parsed as `(method.name === 'SearchProfiles') || 'SearchJobs'` and will be always true, effectively skipping all the methods in Talent API.

Reply:
got it, changes made, and also added unit test case for better coverage.